src/js/control-bar/progress-control/load-progress-bar.js

```diff
@@ -2,7 +2,19 @@
  * @file load-progress-bar.js
  */
 import Component from '../../component.js';
+import {bind} from '../../utils/fn.js';
 import * as Dom from '../../utils/dom.js';
+import activeElement from '../../mixins/active-element.js';
+
+// get the percent width of a time compared to the total end
+const percentify = function(time, end) {
```

Comment:
There isn't a good reason for this to define this every time the update function is run.

Reply:
Note that we have to manually add `%` sign and round now, but this is much more generic.